keep-ui/entities/workflows/model/schema.ts

```diff
@@ -33,6 +33,7 @@ export const V2StepAlertTriggerSchema = z.object({
   properties: z.object({
     alert: AlertTriggerValueSchema,
     source: z.string().optional(),
+    cel: z.string().optional(),
```

Comment:
UI for trigger do not show correct values for valid CEL



https://github.com/user-attachments/assets/e9529b8f-3ecd-4b44-92be-5207cd76bbed

![Screenshot 2025-04-29 at 17 57 17](https://github.com/user-attachments/assets/2ab34dba-b12a-45bc-8a41-0db33c564eae)